addons/bestja_estimation_reports/security/security.xml

```diff
@@ -0,0 +1,116 @@
+<openerp>
+    <data>
+        <record id="estimation_report_own_read_rule" model="ir.rule">
+            <field name="name">Managers / coordinators can see and create their own estimation_reports.</field>
+            <field name="model_id" ref="model_bestja_estimation_report"/>
+            <field name="groups" eval="[(4, ref('base.group_user'))]"/>
+            <field name="domain_force">[
+                '|',
+                    ('project.manager', '=', user.id),
+                    ('project.organization.coordinator', '=', user.id),
+            ]</field>
+            <field name="perm_read" eval="True"/>
+            <field name="perm_write" eval="False"/>
+            <field name="perm_create" eval="True"/>
+            <field name="perm_unlink" eval="False"/>
+        </record>
+
+        <record id="estimation_report_own_modify_rule" model="ir.rule">
+            <field name="name">Managers / coordinators can modify their own estimation reports</field>
+            <field name="model_id" ref="model_bestja_estimation_report"/>
+            <field name="groups" eval="[(4, ref('base.group_user'))]"/>
+            <field name="domain_force">[
+                '|',
+                   ('state', '!=', 'sent'),
```

Comment:
I'd say `('state', '=', 'draft')` to future proof this for situation where other states were to be added.